### PR TITLE
Minor edits to the sparsity RFC

### PR DESCRIPTION
**Pull request by aartbik**
A few edits to make the proposal more consistent with the updated MLIR doc

### CODE REVIEW

File: rfcs/20230210-sparsity.md [hunk 1]
```diff
@@ -142,9 +140,9 @@ The supported level-formats should consist of at least the following:
   for when coordinates have no siblings.
 
 The specification should be easy to extend for adding new level-formats.
-For example, a parameterized type **compressed(2:4)** could be used to
+For example, a parameterized level-format **compressed(2:4)** could be used to
 define 2-out-of-4 structured sparsity that is well-supported by some
-vendors (viz. NVidia GPUs). These three level-formats have two properties:
+vendors (viz. NVidia GPUs). All level-formats have two properties:
 **non/unique** (are duplicates allowed at that level), and
 **un/ordered** (do coordinates appear sorted at that level). Both of
```
[the remark this answers is not in the document]
Comment (aartbik) [reply rfcs/20230210-sparsity.md:147]: Yeah, good catch. Let's change that too.

File: rfcs/20230210-sparsity.md [hunk 2]
```diff
@@ -104,35 +104,33 @@ that was also adopted in MLIR for sparse tensor types. Below we use
...
-optional. (Whereas the level-format for supporting `ITPACK/ELLPACK`
-has no properties and the level-expression is required.)
+optional.
 
-For the three level-types discussed below, each level-expression is an
+For the level-types discussed below, each level-expression is an
 affine expression over dimension-variables. Thus, the level-expressions
 collectively define an affine map from dimension-coordinates to
 level-coordinates. And the dimension-expressions collectively define the
 inverse map— which only needs to be provided for elaborate cases
```
Comment (yinying-lisa-li) [inline rfcs/20230210-sparsity.md:130]: One more nit. Should we remove the - here?

**Review by yinying-lisa-li — APPROVED** [review]

File: rfcs/20230210-sparsity.md (hunk 2 above)
Comment (aartbik) [reply rfcs/20230210-sparsity.md:130]: Yes, we should indeed!

**Review by GleasonK — APPROVED** [review]
It looks like these are minor wording edits / re-arranging, as opposed to a change in technical content from what the previous RFC proposed, as such I don't think that any broader RFC review is needed.

LGTM!

**Comment by aartbik** [comment]
Thanks! Can you merge this for me (we don't seem to have write access)?

**Pull request merged**